dev/devicelab/lib/framework/apk_utils.dart

```diff
@@ -477,3 +522,77 @@ String? validateSnapshotDependency(FlutterProject project, String expectedTarget
   return contentSnapshot.contains('$expectedTarget ')
     ? null : 'Dependency file should have $expectedTarget as target. Instead found $contentSnapshot';
 }
+
+class VersionTuple {
```

Comment:
Then it is especially important to document that this class is a duplicate and link to/from each implementation so that if improvements or fixes are made they can be applied to both. 

This will be easier if the class is in its own file.

Reply:
Moved to own file + added a comment in both files to suggest mirroring changes.